packages/compiler/templates/scaffolding.json

```diff
@@ -145,7 +145,7 @@
   "emitter-ts": {
     "title": "TypeSpec emitter",
     "description": "Create a new package that emits artifacts from TypeSpec.",
-    "compilerVersion": "0.67.2",
+    "compilerVersion": "0.68.0",
```

Comment:
```suggestion
    "compilerVersion": "1.0.0-rc.0",
```

Reply:
ah didn't do the rebuild after changing that